### DISCUSSION TITLE
Add instructions for adding `NSLocationWhenInUseUsageDescription` to `Info.plist` on iOS

### DISCUSSION THREAD

**Discussion by philiplindberg** [rnmapbox/maps#2765] [Ideas]
When using the [iOS installation guide](https://github.com/rnmapbox/maps/blob/main/ios/install.md), I didn't see any instructions for adding the `NSLocationWhenInUseUsageDescription` property to `Info.plist`. I know this is only required when using the `UserLocation` component, but I think it should still be documented as an optional step.

Also, for expo users, since the provided [expo config plugin](https://github.com/rnmapbox/maps/blob/main/plugin/install.md#plugin-configuration) doesn't configure the `NSLocationWhenInUseUsageDescription` property automatically, perhaps linking to a library like [expo-location](https://docs.expo.dev/versions/latest/sdk/location/) (which provides instructions for configuring this property) would be helpful.

**Comment by mfazekas** [comment]
@philiplindberg good idea, can you make a PR for this?

**Comment by philiplindberg** [comment]
Just created a PR: https://github.com/rnmapbox/maps/pull/2766.
Thanks!
